Source: Timo Gurr <tgurr@exherbo.org>
Upstream: Reported, https://bugzilla.gnome.org/show_bug.cgi?id=734131
Reason: fix hardcoded path to arping

diff -Naur NetworkManager-0.9.10.0.old/src/devices/nm-device.c NetworkManager-0.9.10.0/src/devices/nm-device.c
--- NetworkManager-0.9.10.0.old/src/devices/nm-device.c	2014-07-04 02:44:19.000000000 +0200
+++ NetworkManager-0.9.10.0/src/devices/nm-device.c	2014-09-01 17:35:29.656282068 +0200
@@ -4429,7 +4429,7 @@
 static void
 send_arps (NMDevice *self, const char *mode_arg)
 {
-	const char *argv[] = { "/sbin/arping", mode_arg, "-q", "-I", nm_device_get_ip_iface (self), "-c", "1", NULL, NULL };
+	const char *argv[] = { "/bin/arping", mode_arg, "-q", "-I", nm_device_get_ip_iface (self), "-c", "1", NULL, NULL };
 	int ip_arg = G_N_ELEMENTS (argv) - 2;
 	NMConnection *connection;
 	NMSettingIP4Config *s_ip4;
